composer.json

```diff
@@ -21,14 +21,13 @@
     "extra": {
     },
     "require": {
-        "php": "^7.3 || ~8.0.0",
-        "laminas/laminas-escaper": "^2.5",
-        "laminas/laminas-validator": "^2.10",
-        "laminas/laminas-zendframework-bridge": "^1.0"
+        "php": "^7.3 || ~8.0.0 || ~8.1.0",
+        "laminas/laminas-escaper": "^2.9",
+        "laminas/laminas-validator": "^2.15"
     },
     "require-dev": {
-        "laminas/laminas-coding-standard": "^2.1",
-        "phpunit/phpunit": "^9.3"
+        "laminas/laminas-coding-standard": "~2.2.1",
```

Comment:
Is this held back on purpose? Considering we have a `composer.lock`, it should stay as `^2.1`?

Reply:
@weierophinney want to change it, or should we defer?

Reply:
Rules change in new minor versions, which means that a contributor running `composer update` suddenly becomes responsible for CS changes that are not about code they've changed. We decided some time ago that we should be using the `~` operator for this dependency to prevent those scenarios.

Reply:
Works for me.

Reply:
For instance... the 2.3 series adds `declare(strict_types=1)` to the top of every file, which can have BC ramifications in some instances.

Reply:
Yeah, I think we need to be more careful with `laminas/laminas-coding-standard` then